scilpy/reconst/frf.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+import logging
+import numpy as np
+from dipy.core.gradients import gradient_table
+from dipy.reconst.csdeconv import auto_response
+from dipy.segment.mask import applymask
+
+from scilpy.utils.bvec_bval_tools import (is_normalized_bvecs, normalize_bvecs,
+                                          check_b0_threshold)
+
+
+def compute_ssft_frf(data, bvals, bvecs, mask=None, mask_wm=None,
```

Comment:
In fact, should be `ssst`: single shell single tissue.

Reply:
Woops, fixed